executor/src/test/java/org/k8loud/executor/service/MapperServiceImplTest.java

```diff
@@ -0,0 +1,81 @@
+package org.k8loud.executor.service;
+
+import data.ExecutionRQ;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.k8loud.executor.action.Action;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class MapperServiceImplTest {
+    private MapperServiceImpl mapperService;
+    private ExecutionRQ executionRQ;
+
+    @BeforeEach
+    public void beforeEach() {
+        mapperService = new MapperServiceImpl();
```

Comment:
could be in `@BeforeAll`

Reply:
We want to make sure that each test is run separately.
If `MapperServiceImpl` was stateful (which it shouldn't be), it could affect subsequent tests.

Reply:
It is a service so tbh it shouldn't be either in BeforeEach. Perhaps we should use @Autowired ?